guides/source/rails_on_rack.md

```diff
@@ -80,6 +80,18 @@ To find out more about different `rackup` options, you can run:
 $ rackup --help
 ```
 
+#### Prefix your app using `config.relative_url_root`
+
+When using `rackup` and you are deploying your app to a subdirectory, you need to use `map` to define the root url to your app when generating the `Rack::URLMap` structure.
```

Comment:
Is `rackup` correct here? Pretty sure that puma reads config.ru and possibly others?

Reply:
@pixeltrix Good point, I was using this in the context of a rackup file, would it make more sense to just say "When using `puma` or other Ruby web servers that use  the rackup file `config.ru`..."?

Reply:
Also maybe Rails on Rack guide isn't the best place 🤔